data/json/effects_on_condition/medicine_eocs/regen_serum_eoc.json

```diff
@@ -0,0 +1,97 @@
+[
+  {
+    "type": "jmath_function",
+    "id": "spent_to_ingested_ratio",
+    "//": "return ratio between calories you spent yesterday and calories you ingested yesterday. if bigger than 1, you consumed more calories than you ingested",
+    "num_args": 0,
+    "return": "max( get_calories_daily('type':'spent', 'day':'1'), 1 ) / max( get_calories_daily('type':'ingested', 'day':'1'), 1 )"
+  },
+  {
```

Comment:
```suggestion
  {
    "type": "effect_on_condition",
    "id": "EOC_SERUM_GENERIFIED",
    "condition": { "expects_vars": [ "serum_duration" ] },
    "effect": { "run_eocs": "EOC_SERUM_GENERIFIED_2" },
    "false_effect": { "u_message": "This is a bug.  serum_duration needs to be defined in serums.json.", "type": "bad" }
  },
  {
    "type": "effect_on_condition",
    "id": "EOC_SERUM_GENERIFIED_2",
    "condition": { "or": [ { "u_has_item": "syringe" }, { "u_has_bionics": "bio_syringe" } ] },
    "effect": [ { "run_eocs": "EOC_SERUM_GENERIFIED_3" } ],
    "false_effect": { "u_message": "You need a syringe to inject the liquid.", "type": "bad" }
  },
  {
    "type": "effect_on_condition",
    "id": "EOC_SERUM_GENERIFIED_3",
    "condition": { "u_has_any_effect": [ "regen_serum_main", "regen_serum_crash_cardio", "regen_serum_crash_fatigue" ] },
    "effect": { "u_message": "You would not dare to inject two of them at once.", "type": "bad" },
    "false_effect": [ { "u_message": "You inject the liquid, with hope it won't kill you." }, { "run_eocs": "EOC_SERUM_REGEN" } ]
  },
  {
```
Let's see if these can help solving the error. If it works, then serum_eoc.json is not needed and should be deleted.

Reply:
I plan to reuse infra for another serums, once there would be a way to add them, that's the reason `serum_eoc` was created
i'll check this one locally, when i'll have some time